src/routes/authors.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const { 
+   createAuthor, 
+   readAuthors, 
+   getAuthorById, 
+   updateAuthor, 
+   deleteAuthor 
+} = require('../controllers/authors');
+
+const authorRouter = express.Router();
+
+authorRouter.route('/')
+.post(createAuthor)
+.get(readAuthors);
+
+authorRouter.route('/:id')
+.get(getAuthorById)
+.patch(updateAuthor)
+.delete(deleteAuthor);
```

Comment:
For longer chained commands like this, you should use one method per line and also indent each one so it can quickly be seen that they all belong to the same parent
```suggestion
authorRouter.route('/:id')
  .get(getAuthorById)
  .patch(updateAuthor)
  .delete(deleteAuthor);
```